<launch>
    <arg name="fps" default="5" />
    <arg name="model" default="$(find neuro_car)/models/fatigue.model" />
    <arg name="topic" default="fatigue" />

    <node name="svm_fatigue" pkg="neuro_car" type="svm" output="screen">
        <param name="fps" type="int" value="$(arg fps)" />
        <param name="model" type="string" value="$(arg model)" />
        <param name="topic" type="string" value="$(arg topic)" />
    </node>
</launch>
